docs/docs/deep-dive/caching-events.md

```diff
@@ -44,7 +44,7 @@ sudo ./dist/tracee \
     -o format:json \
     -o option:parse-arguments \
     -trace container \
-    --cri docker:/var/run/docker.sock
+    --crs docker:/var/run/docker.sock
```

Comment:
Did you brought main upstream onto your branch? The crs->cri is a recent change.

Reply:
yes, it got a bit messed up but I think I fixed it in my PR